borg/archiver.py

```diff
@@ -112,7 +113,7 @@ def print_warning(self, msg, *args):
 
     def print_file_status(self, status, path):
         if self.output_list and (self.output_filter is None or status in self.output_filter):
-            logger.info("%1s %s", status, remove_surrogates(path))
+            logger.getLogger('borg.output.list').info("%1s %s", status, remove_surrogates(path))
```

Comment:
hmm, the logger object is a way to lazily do logging.getLogger(modulename).
calling this with another .getLogger(absolute name) seems a bit strange.
can this be done more easily?